### PR TITLE
Initial theme styling 1

### PR DESCRIPTION
**Pull request by typaulhus**
- scss structure
- added blog posts from appland.com (this can be hidden if necessary)
- appland variables & styles added
- Documentation page placeholder

Screen Shot 2020-10-14 at 11 13 17 AM

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -23,14 +23,16 @@ email: your-email@example.com
 description: >- # this means to ignore newlines until "baseurl:"
   Describe software architecture with data
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site, e.g. http://example.com
-twitter_username: jekyllrb
-github_username:  jekyll
+url: 'https://appmaps.org'
```
Comment (kgilpin) [inline _config.yml:26]: Should be `appland.org`

File: _includes/navigation.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<nav>
+  <a class="logo" href="/">AppMap</a>
```
Comment (kgilpin) [inline _includes/navigation.html:2]: I think we have decided this should be `AppLand.org`

Comment (Daniel-Warner-X) [reply _includes/navigation.html]: Updated

Comment (typaulhus) [reply _includes/navigation.html]: Ill change the page heading as well to match. :)

**Pull request merged**